scripts/vm/systemvm/injectkeys.sh

```diff
@@ -27,7 +27,7 @@ set -e
 TMP=/tmp
 MOUNTPATH=${HOME}/systemvm_mnt
 TMPDIR=${TMP}/cloud/systemvm
-
+umask 022
```

Comment:
Note: This simply affects only those files created by this script's owner user and not by root (or sudo).